yorkie/src/main/kotlin/dev/yorkie/document/crdt/Primitive.kt

```diff
@@ -0,0 +1,31 @@
+package dev.yorkie.document.crdt
+
+import dev.yorkie.document.time.TimeTicket
+
+internal class Primitive(
+    val value: Any?,
+    val createdAtTime: TimeTicket,
+) : CrdtElement(createdAtTime) {
+    var type: PrimitiveType? = null
+
+    init {
+        type = when (value) {
+            is Boolean -> PrimitiveType.Boolean
+            is Int -> PrimitiveType.Integer
+            is Long -> PrimitiveType.Long
+            is Double -> PrimitiveType.Double
+            is String -> PrimitiveType.String
+            is ByteArray -> PrimitiveType.Bytes
+            // TODO support Date type
```

Comment:
@hackerwins should we write `TODO` in the form below?
```suggestion
TODO(daeyoungInc): support Date type
```
(ref: https://github.com/yorkie-team/yorkie-android-sdk/pull/7#discussion_r972191488)